src/styles/index.css

```
@@ -0,0 +1,7 @@
+@import "./reset.css";
+
+@import "./block/card.css";
+
+@import "./block/stars.css";
+
+@import "./block/page.css";
```

Empty lines are redundant here
```suggestion
@import "./reset.css";

@import "./block/card.css";
@import "./block/stars.css";
@import "./block/page.css";
```